### ISSUE TITLE
username atrribute force to SamAccountName

### ISSUE THREAD

**Issue by anetrjuns** [abbas-gheydi/radotp#9]
How to force to use SamAccountName instead of UserPrincipalName?

**Comment by anetrjuns** [comment]
It's not working when   ForceSearchForSamAccountName = true enabled


```
2024/05/31 12:08:07 received a radius packet for user: arsen.tonoyan
2024/05/31 12:08:07 Access-Reject to 10.16.4.48:34207 for arsen.tonoyan stage active directory
```

**Comment by abbas-gheydi** [comment]
How many dot (.) characters are in your domain name?

**Comment by anetrjuns** [comment]
only one 
```
domain.am
```

**Comment by anetrjuns** [comment]
basedn is 
```
basedn = "DC=ad,DC=domain,DC=am"
```

**Comment by abbas-gheydi** [comment]
This issue is fixed in version 2.3.0.

Thanks, dear @anetrjuns , for reporting this issue.
